kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -502,6 +502,27 @@ object KyuubiConf {
       .timeConf
       .createWithDefault(Duration.ofSeconds(5).toMillis)
 
+  val OPERATION_FORCE_CANCEL: ConfigEntry[Boolean] =
+    buildConf("operation.interrupt.on.cancel")
+      .doc("When true, all running tasks will be interrupted if one cancels a query. " +
+        "When false, all running tasks will remain until finished.")
+      .version("1.2.0")
+      .booleanConf
+      .createWithDefault(true)
+
+  val OPERATION_QUERY_TIMEOUT: ConfigEntry[Long] =
+    buildConf("operation.query.timeout")
```

Comment:
Decide to hide the Spark thing that in `SparkThriftServer,` so define the config with Kyuubi style @pan3793

Reply:
I think Kyuubi style timeConf defined in ms? The comments here is in seconds.

Reply:
Actually, I think we don't need specific time unit in kyuubi timeConf style, user should not care about what we use internal, they just define it via ISO-8601 duration string

Reply:
Sorry, I missed this comment. What @pan3793 pointed out is true, we don't need to tell the unit of a time conf, it's Duration  ISO-8601 strings..

Here is a bug at caller side! the client timeout is conf, and the system timeout is ms... @ulysses-you